### PR TITLE
Fix/48 Add color palette guidelines

### PR DESCRIPTION
**Pull request by D-Rosa99**

### CODE REVIEW

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
```
Comment (iocedano) [inline android/gradle/wrapper/gradle-wrapper.properties]: Please don't update the version now. we need to do more test!

File: app/constants/colors.js [hunk 2]
```diff
@@ -61,6 +63,26 @@ const colors = {
...
+  noticeText: '#fff',
+  orange: '#FEB313',
+  pink: '#F1485B',
+  purple: '#525BD9',
+  tabBar: '#fefefe',
+  tabIconDefault: '#ccc',
+  tabIconSelected: tintColor,
+  tintColor,
+  warningBackground: '#EAEB5E',
+  warningText: '#666804',
```
Comment (iocedano) [inline app/constants/colors.js]: Let's be consistent with the naming convention.
Could you change those names to uppercase?

File: app/constants/colors.js [hunk 3]
```diff
@@ -61,6 +63,26 @@ const colors = {
...
+  buttonLightText: '#6B6B6B',
+  noticeBackground: tintColor,
+  noticeText: '#fff',
+  orange: '#FEB313',
+  pink: '#F1485B',
+  purple: '#525BD9',
+  tabBar: '#fefefe',
+  tabIconDefault: '#ccc',
+  tabIconSelected: tintColor,
+  tintColor,
```
Comment (iocedano) [inline app/constants/colors.js]: Could we just use one variable for this?

File: app/constants/colors.js (hunk 2 above)
Comment (iocedano) [reply app/constants/colors.js]: also, Could we named as the color name not base on what we use the color?
example:
```js
warningText: '#666804'
```
to 
```js
YUKON_GOLD: '#666804'
```

http://chir.ag/projects/name-that-color/

**Review by iocedano — CHANGES_REQUESTED** [review]

File: app/constants/themes.js [hunk 4]
```diff
@@ -4,17 +4,19 @@ import React from 'react';
 
 import Color from './colors';
 
+// import { Colors } from 'react-native/Libraries/NewAppScreen';
```
Comment (RutsSantos) [inline app/constants/themes.js]: Remove comment

File: app/constants/themes.js [hunk 5]
```diff
@@ -4,17 +4,19 @@ import React from 'react';
...
+
 /** Violet on pale violet bg. e.g. Settings */
 export const defaultTheme = {
-  background: Color.VIOLET_ALPHA_06,
-  textPrimaryOnBackground: Color.VIOLET,
-  textSecondaryOnBackground: 'rgba(64, 81, 219, 0.6)',
+  background: Color.lightBlue,
```
Comment (RutsSantos) [inline app/constants/themes.js]: "lightBlue" in Uppercase

File: app/views/onboarding/Onboarding1.js [hunk 6]
```diff
@@ -20,6 +20,7 @@ import { EulaModal } from '../../components/EulaModal';
 import NativePicker from '../../components/NativePicker';
 import { Typography } from '../../components/Typography';
 import Colors from '../../constants/colors';
+import colors from '../../constants/colors';
```
Comment (RutsSantos) [inline app/views/onboarding/Onboarding1.js]: Why do you have two imports of the same thing?

**Review by RutsSantos — COMMENTED** [review]
Left some comments

**Review by adnanjt — APPROVED** [review]
LGTM

File: app/constants/colors.js [hunk 7]
```diff
@@ -8,7 +10,7 @@ const colors = {
   MISCHKA: '#E5E4E6',
   PRIMARY_TEXT: '#000',
   SUCCESS: '#41dca4',
-  GREEN: '#41dca4',
+  GREEN: '#7ED321',
   WARNING: '#ffc000',
   orange: '#ffc000',
```
Comment (inteljudajocu) [inline app/constants/colors.js]: change `orange` to `ORANGE`

File: app/views/Import.js [hunk 8]
```diff
@@ -124,23 +124,23 @@ const styles = StyleSheet.create({
...
-  },
-  buttonText: {
-    fontFamily: fontFamily.primarySemiBold,
-    fontSize: 16,
-    lineHeight: 19,
-    letterSpacing: 0,
-    textAlign: 'center',
-    color: colors.WHITE,
-  },
+  // buttonTouchable: {
```
Comment (inteljudajocu) [inline app/views/Import.js]: Remove comments if unnecessary

**Review by inteljudajocu — COMMENTED** [review]
Left comments

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**